### PR TITLE
[Feature] Add basic pwa

### LINKED ISSUE #112: PWA???🤔🤷‍♂️

### PR DESCRIPTION
**Pull request by conalli**
## Description

Add basic PWA functionality (can install standalone version) and update icons/favicon

## Closes issue(s)

Closes #112 #119 

## Screenshots

Screenshot 2021-09-14 at 14 03 52
Screenshot 2021-09-14 at 14 05 26
Screenshot 2021-09-14 at 14 08 17

Standalone version (Title is no longer set to React App but browser is using cached version I think)
Screenshot 2021-09-14 at 14 08 38

## Changes include

- [x] New feature (non-breaking change that adds functionality)

## Checklist

- [x] I have tested this code

### CODE REVIEW

**Review by kome12 — APPROVED** [review]
Thanks for taking care of this! LGTM!

**Review by curtisahuang — APPROVED** [review]
LGTM

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
+            "src": "/icon-384x384.png",
+            "sizes": "384x384",
+            "type": "image/png"
+        },
+        {
+            "src": "/icon-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ]
```
Comment (daiHash) [inline public/manifest.json:31]: 👍

**Review by daiHash — APPROVED** [review]
lgtm

**Pull request merged**